rusk-recovery/src/bin/state.rs

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+mod task;
+mod version;
+
+use clap::Parser;
+use std::path::PathBuf;
+use version::VERSION_BUILD;
+
+#[derive(Parser, Debug)]
+#[clap(name = "rusk-recovery-state")]
+#[clap(author, version = &VERSION_BUILD[..], about, long_about = None)]
+struct Cli {
+    /// Sets the profile path
+    #[clap(
```

Comment:
You can always use `#[clap(flatten)]` if you want to only define `RUSK_PROFILE_PATH` once. The benefits are questionable.